lib/models/badge.dart

```diff
@@ -26,8 +34,19 @@ class Badge {
       case 'SUPER_FOUNDER': return BadgeKeyword.super_founder; break;
       default: return BadgeKeyword.none; break;
     }
+  }
 
+  static String _getBadgeKeywordFromEnum(enumKeyword) {
```

Comment:
Check how Jakob did the OBNotificationsType, he's using a class to do this really nicely.

Reply:
👍